python/hsfs/feature_group.py

```diff
@@ -48,7 +48,8 @@
 
 
 class FeatureGroupBase:
-    def __init__(self, featurestore_id, location):
+    def __init__(self, featurestore_id, location, event_time=None):
+        self.event_time = event_time
```

Comment:
Can we make this private and have getters/setters?

Reply:
It is assigning using setter now.